src/common/fs/FileUtils.cpp

```diff
@@ -364,6 +364,14 @@ bool FileUtils::exist(const std::string& path) {
     return access(path.c_str(), F_OK) == 0;
 }
 
+// static
+bool FileUtils::rename(const std::string& src, const std::string& dst) {
+    auto status = ::rename(src.c_str(), dst.c_str());
+    LOG_IF(WARNING, status < 0) << "Rename " << src << " to " << dst << " failed, the errno: "
```

Comment:
Is betterLOG_IF(ERROR, status != 0) ?

Reply:
Seems the negative integer represent error cases.

Reply:
Aha，Suppose you are right，Do return value need to be uniform?  
` return status == 0;`
The actual：
```
If the file is successfully renamed, a zero value is returned.
On failure, a nonzero value is returned.
```

Reply:
The others looks good to me. well done.

Reply:
Good point.